Source/Extensions/Blazorise.RichTextEdit/RichTextEditToolbarSelectItem.razor

```diff
@@ -0,0 +1 @@
+﻿<option @attributes=attributes>@ChildContent</option>
```

Comment:
Why attributes and not Selected and Value properties instead?

Reply:
I thought that didn't work but when I try it now it does so removed the attributes